metpy/calc/tests/test_tools.py

```diff
@@ -14,7 +14,8 @@
                         nearest_intersection_idx, parse_angle, pressure_to_height_std,
                         reduce_point_density, resample_nn_1d, second_derivative)
 from metpy.calc.tools import (_get_bound_pressure_height, _greater_or_close, _less_or_close,
-                              _next_non_masked_element, delete_masked_points, DIR_STRS)
+                              _next_non_masked_element, delete_masked_points, DIR_STRS,
+                              gradient)
```

Comment:
The stated purpose of #712 is to catch the fact that `gradient` is not exported in the `calc` API. So you’ll need to change this import and then fix it. 😁

Reply:
Ah I misunderstood the issue wording "So the reason we were able to ship without exporting gradient is because there are no explicit tests for it." I see

Reply:
So what you’re saying is that I don’t make good bug reports... 😁